5.5/test/run

```diff
@@ -214,22 +214,29 @@ function run_container_creation_tests() {
   echo "  Success!"
 }
 
-function test_config_option() {
-  local env_var=$1 ; shift
-  local setting=$1 ; shift
-  local value=$1 ; shift
-
-  # If $value is a string, it needs to be in simple quotes ''.
-  # If nothing is found, grep returns 1 and test fails.
-  docker run --rm -e $env_var=${value//\'/} $IMAGE_NAME cat /var/lib/mysql/my.cnf | grep -q "$setting = $value"
-}
-
 function run_configuration_tests() {
   echo "  Testing image configuration settings"
-  test_config_option MYSQL_LOWER_CASE_TABLE_NAMES lower_case_table_names 1
-  test_config_option MYSQL_MAX_CONNECTIONS max_connections 1337
-  test_config_option MYSQL_FT_MIN_WORD_LEN ft_min_word_len 8
-  test_config_option MYSQL_FT_MAX_WORD_LEN ft_max_word_len 15
+
+  create_container \
+    config_test \
+    --name config_test \
+    --env MYSQL_ROOT_PASSWORD=config_test \
+    --env MYSQL_LOWER_CASE_TABLE_NAMES=1 \
+    --env MYSQL_MAX_CONNECTIONS=1337 \
+    --env MYSQL_FT_MIN_WORD_LEN=8 \
+    --env MYSQL_FT_MAX_WORD_LEN=15 \
```

Comment:
Yep, you know this trick, right? Next time diff will be less, because I will change one line. If you want I can remove it (maybe even in next PR where I will add tests for new parameters).